chroma_core/models/stratagem.py

```diff
@@ -286,24 +337,16 @@ def get_body(mount_point, report_duration, purge_duration):
                         "rules": [
                             {
                                 "action": "LAT_COUNTER_INC",
-                                "expression": "&& < size 1048576 != type S_IFDIR",
-                                "argument": "SIZE < 1M",
-                            },
-                            {
-                                "action": "LAT_COUNTER_INC",
-                                "expression": "&& >= size 1048576000000 != type S_IFDIR",
+                                "expression": ">= size 1048576000000",
```

Comment:
I can put this into a separate patch if you guys would like. We were not picking up files that were >= 1tb. This change fixes that issue.

Reply:
It's already landed in a separate patch.

Reply:
Ok cool, i'll remove this then.